### ISSUE TITLE
No response with Nano Connect or PicoW

### ISSUE THREAD

**Issue by bmck202** [Bodmer/OpenWeather#12]
I do have a problem getting a sketch to work with the Nano Connect. I am trying to run Bodmer’s Openweather LittleFS and it says download complete and all I get is a black screen. I am running TFT_eSPI setup 62. I can run thru all examples with no issues. I then load OpenWeather Test and WiFi responds and open weather links up and displays serial data. I then try to run OpenweatherLittleFS and all I get is download complete with the file size and no response in serial monitor and TFT display. I am running Arduino IDE 2.1 with a MacBook pro with 13.3. I can’t help but think I have missed enabling something but do not have enough experience to know what. Any thoughts????

**Comment by Bodmer** [comment]
Have you created a LittleFS partition and uploaded the data?

**Comment by bmck202** [comment]
I have not. Is this something close to what you are saying..https://github.com/littlefs-project
/
[littlefs](https://github.com/littlefs-project/littlefs)??

**Comment by Bodmer** [comment]
LittleFS is part of Earle Philhower's RP2040 board support package. Are you using that package? The Arduino mbed package does not provide LittleFS capability.

**Comment by bmck202** [comment]
I did download and use Earle Philhower's board support package. Is the Littlefs download in the package?

**Comment by Bodmer** [comment]
https://github.com/Bodmer/OpenWeather/blob/main/examples/TFT_eSPI_OpenWeather_LittleFS/All_Settings.h#L24

**Comment by Bodmer** [comment]
https://arduino-pico.readthedocs.io/en/latest/fs.html#uploading-files-to-the-littlefs-file-system

**Comment by bmck202** [comment]
Do I add the plugin in the same manner as the esp32 and esp8266 plugins on my Mac and will it work with IDE2.1?

**Comment by bmck202** [comment]
My apologies for being a complete noob but my electronics training ended 48 years ago and most everything was vacuum tubes. I am using 2.1 because i am also having issues getting the plugins installed on my MacBook.

**Comment by Bodmer** [comment]
Good point, I have not tried the LittleFS plugin with IDE 2.x I use 1.8.19

**Comment by bmck202** [comment]
I have reinstalled 1.8.19 on my Mac. I also uninstalled 2.1, they don't play well together on the Mac. I tried the install for plugins per the Mac OS install instructions and still no data load in tools. I was hoping to get around the plugins by using the RP2040 or else I would have already been setup on esp32.

**Comment by Bodmer** [comment]
There seem to be specific instructions for the Mac OS, see here:
https://github.com/earlephilhower/arduino-pico-littlefs-plugin
 Note that different install folders are mentioned.

**Comment by Bodmer** [comment]
I have found this:
https://github.com/earlephilhower/arduino-pico-littlefs-plugin/issues/3

So it looks like IDE 2.x does hot handle plugins yet. So 1.8.x is the only option. I am not a Mac OS user so am unable to help.

Without a means to upload the required image files the LittleFS example will not work. The only other option is to adapt the example to use an SD card but this is not something I can help with.

Since there is no reason to suspect this library code and my tests show it is working, then I will close the issue.

I hope you find a solution. Maybe raise the issue with Earl Philhower as he is very helpful. If you do find a solution, then post back and I can add this to the ReadMe.

**Comment by bmck202** [comment]
Thank you for all your help. If I have 1.8.19 and 2.1 loaded on my Mac at the same time, the Mac prioritizes 1.8 and tries to open the .h files as an ino. Because of this, I am unable to make my selections in user setup. My other option my be to obtain a Windows platform pc and try that(seems most people use that anyway). Anyway, Thanks again and I will keep trying....

**Comment by bmck202** [comment]
Update.... I did figure out what I was doing wrong on the data loader install. If I had read the instructions, I would have known to download the release notes zip and install it. I got Spiffs openweather on ESP32C3 and ESp32S3 without issues...Thank you very much. The Rp2040s are a different matter. I download data files are get all files installed. I check file list and all are loaded but one (skipping DS_store). I download sketch and in serial print I see Littlefs not found dir corrupt 0x0 0x1
